static/admin/config.yml

```diff
@@ -31,6 +31,7 @@ collections:
     fields:
       - {label: "Template Key", name: "templateKey", widget: "hidden", default: "design-guideline-post"}
       - {label: "Title", name: "title", widget: "string"}
+      - {label: "Featured Image", name: "featuredImage", widget: "image", tagname: ''}
```

Comment:
Should add these as required attributes.

Reply:
If I have seen this right, required is true by default (at least I couldn't save the page without selecting an image), but I'll check this again

Reply:
![image](https://user-images.githubusercontent.com/8302493/51900618-b71dab00-23ad-11e9-915d-c28f1e64cf11.png)
from [here](https://www.netlifycms.org/docs/configuration-options/), required defaults to true

Reply:
Great. Thanks 👍

Reply:
https://github.com/devinea/mvx19-cms/pull/26/files#diff-e36160419abb71f07eb8f74e9d3cc8bcL21 do you want this to be required as well? (the featuredImage in the blog)

Reply:
Yes.

Reply:
Okay, will be adjusted :)